### PR TITLE
Splitted Admin API reference into multiple pages

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Split the "Admin API" reference into multiple pages and promote it to its own sections in the left website sidebar.

cc/ @lucperkins

### CODE REVIEW

**Comment by merlimat** [comment]
@rdhabalia Can you take a look at this one too? This is the change we had talked on the slack discussion

File: site/docs/latest/admin-api/partitioned-topics.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
+---
+title: Partitioned topics
```
Comment (rdhabalia) [inline site/docs/latest/admin-api/partitioned-topics.md]: `Managing Partitioned topics`? in other doc, we have started with `Managing`.

File: site/docs/latest/admin-api/permissions.md [hunk 2]
```diff
@@ -1,155 +1,6 @@
...
-#### Java
-
-```java
-admin.namespaces().deleteNamespace(namespace);
-```
+---
+title: Managing permissions
+---
 
 ## Managing permissions
```
Comment (rdhabalia) [inline site/docs/latest/admin-api/permissions.md]: as title name is `Managing permissions`, should we just write `permissions` here?

File: site/docs/latest/admin-api/permissions.md [hunk 3]
```diff
@@ -1,155 +1,6 @@
...
-[More info](../../reference/RestApi#/admin/namespaces/:property/:cluster/:namespace)
-
-#### Java
-
-```java
-admin.namespaces().deleteNamespace(namespace);
-```
+---
+title: Managing permissions
+---
```
Comment (rdhabalia) [inline site/docs/latest/admin-api/permissions.md]: actually, if you see the content of this file: it has `set replication cluster`, `split-bundle`, `unload-bundle`,.. it doesn't seem like a permissions page? Do we really need this page, or we can keep this content into namespaces.md only?

File: site/docs/latest/admin-api/partitioned-topics.md (hunk 1 above)
Comment (merlimat) [inline site/docs/latest/admin-api/partitioned-topics.md]: Yes, fixed

File: site/docs/latest/admin-api/permissions.md (hunk 2 above)
Comment (merlimat) [inline site/docs/latest/admin-api/permissions.md]: I'll change the title below (` ## Managing permissions`) to keep it consistent with the other admin api pages

File: site/docs/latest/admin-api/permissions.md (hunk 3 above)
Comment (merlimat) [inline site/docs/latest/admin-api/permissions.md]: You're right, I broke it down in the wrong place. Put all these operations under namespace again.

**Comment by merlimat** [comment]
@rdhabalia  updated

**Comment by merlimat** [comment]
retest this please

**Pull request merged**